src/js/ResponsiveNav.js

```diff
@@ -55,16 +57,57 @@ function ResponsiveNav(rootEl) {
 		moreListEl.appendChild(itemEl);
 	}
 
+	function cloneItemToMoreList(el) {
+		var cloneEl = el.cloneNode(true);
+		// remove the attributes that are only applicable to higher level
+		cloneEl.removeAttribute('data-priority');
+		cloneEl.removeAttribute('aria-hidden');
+		cloneEl.removeAttribute('data-o-hierarchical-nav--is-cloneable');
+		// recurse through children and amend any id values to maintain uniqueness
+		prefixIds(el);
+		moreListEl.appendChild(cloneEl);
+	}
+
+	function resetIds() {
+		var nextNode;
+		while (prefixedNodes.length > 0) {
+			nextNode = prefixedNodes.pop();
+			nextNode.setAttribute('id', nextNode.getAttribute('id').replace(clonedIdPrefix, ''));
+		}
+	}
+
+	function prefixIds(el) {
+		var child;
+		if (el.hasChildNodes()) {
+			var children = el.childNodes;
+			for (var i = 0, l = children.length; i < l; i++) {
+				child = children[i];
+				if (child instanceof HTMLElement) {
+					if (child.hasAttribute('id')) {
+						prefixedNodes.push(child); // store to make the cleanup more performant
+						child.setAttribute('id', clonedIdPrefix + child.getAttribute('id'));
+					}
+					prefixIds(child);
+				}
+			}
+		}
+	}
+
 	// For every hidden item, add it to the more list
 	function populateMoreList(hiddenEls) {
 		emptyMoreList();
+		resetIds();
 
 		for (var c = 0, l = hiddenEls.length; c < l; c++) {
 			var aEl = hiddenEls[c].querySelector('a');
 			var ulEl = hiddenEls[c].querySelector('ul');
 
-			var aText = (typeof aEl.textContent !== 'undefined') ? aEl.textContent : aEl.innerText;
-			addItemToMoreList(aText, aEl.href, ulEl);
+			if (hiddenEls[c].hasAttribute('data-o-hierarchical-nav--is-cloneable')) {
```

Comment:
should be data-o-hierarchical-nav-is-cloneable, we only use BEM like syntax in classes